### PR TITLE
Milestone: Factory

### LINKED ISSUE #2800: Merge milestone 'Factory' to Develop
## Milestone completion: Factory

All issues in milestone 'Factory' are now complete.
This is an auto-generated tracking issue for the milestone to Develop merge.

### Closed issues
- #2794: Creature Factory: seed a smarter initial creature from observations, outputs & cost (optional training-data scan)
- #2793: Couple output activation + backprop loss to the cost function (add softmax; cross-entropy gradient for classification)
- #2792: Weak structural growth under coarse argmax CATEGORICAL_ERROR fitness
- #2791: trainPerGen default of 1 starves supervised gradient training
- #2780: Workflow best-practice findings

### Action required
Raise a PR to merge the milestone branch to `Develop`.

### PR DESCRIPTION
**Pull request by VibeCoderST**
## Milestone: Factory

This PR merges the completed milestone branch into Develop.
Closes #2800

### Issues addressed
- #2794: Creature Factory: seed a smarter initial creature from observations, outputs & cost (optional training-data scan)
- #2793: Couple output activation + backprop loss to the cost function (add softmax; cross-entropy gradient for classification)
- #2792: Weak structural growth under coarse argmax CATEGORICAL_ERROR fitness
- #2791: trainPerGen default of 1 starves supervised gradient training
- #2780: Workflow best-practice findings

### Review notes
All individual issues were reviewed and merged into the milestone branch via separate PRs.
This final PR consolidates all changes for a comprehensive review before merging to Develop.

### CODE REVIEW

**Comment by nleck** [comment]
Please fix the spelling issues 

src/architecture/CreatureFactory.ts:158:36 Unknown word (Glorot)
test/methods/activations/SOFTMAX.ts:78:9 Unknown word (probs)
test/methods/activations/SOFTMAX.ts:80:19 Unknown word (probs)
test/methods/activations/SOFTMAX.ts:91:9 Unknown word (probs)
test/methods/activations/SOFTMAX.ts:93:19 Unknown word (probs)
test/methods/activations/SOFTMAX.ts:99:10 Unknown word (probs)
Warning: Unknown word (probs)
Warning: Unknown word (probs)
Warning: Unknown word (probs)
Warning: Unknown word (probs)
Warning: Unknown word (probs)

**Comment by VibeCoderST** [comment]
I've pushed fixes for the spelling issues. Please review the changes.

**Comment by VibeCoderST** [comment]
Claiming PR feedback comment 4572009248 for worker `GRQ-23-68298`.

**Comment by VibeCoderST** [comment]
I reviewed this feedback and could not identify a code change to apply. Please clarify if a change is required.

**Comment by VibeCoderST** [comment]
## Coverage CI Fix

The `coverage` check was failing due to a test failure in `test/docs/JekyllLiquidSafety.ts`, not a coverage threshold issue.

### Root Cause

`docs/archive/pr-summaries/pr-summary-2780.md` (line 26) contained a bare `${{ secrets.ACTIONS_PUSH || secrets.GITHUB_TOKEN }}` expression inside a fenced YAML code block. Jekyll's Liquid templating engine processes `{{` sequences in raw source **before** Markdown fences are interpreted, so the fenced block provided no protection.

### Fix

Wrapped the fenced code block in `{% raw %}` / `{% endraw %}` tags in `docs/archive/pr-summaries/pr-summary-2780.md`. This is the standard escape for Liquid syntax in Jekyll documentation.

### Verification

Ran `deno test -A test/docs/JekyllLiquidSafety.ts` locally — all 7 tests pass (0 failures).

Committed as `8ba00892` on `milestone/factory`.

**Review by nleck — APPROVED** [review]
